client/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import { Routes,Route } from 'react-router-dom';
```

Comment:
Use a formatter for indentation. I prefer ESlint + Prettier. But ESlint can mess up the code, so beware about that. Here  are the docs - [ESLint](https://eslint.org/docs/user-guide/getting-started) & [Prettier](https://prettier.io/docs/en/index.html).


Check especially this part for prettier [plugin](https://prettier.io/docs/en/integrating-with-linters.html) if you're planning on using both ESLint & Prettier